ppbkc_generateBaseData.m

```diff
@@ -23,6 +23,8 @@
 machine.data.energy         = params('photon_energy');
 machine.data.kernelPos      = [0:0.5:179.5];
 machine.data.fwhm           = params('fwhm_gauss');
+machine.data.iNumKernel     = params('i_num_kernel');
+machine.data.electronRangeIntensity     = params('electron_range_intensity');
```

Comment:
What do we need this parameter for? Is it used later in the dose calculation? Here for the kernels it seems superfluos for the subsequent calculations...

Reply:
The additional parameters are needed for dose calculation in matRad.